src/plugins/intel_cpu/src/nodes/rdft.cpp

```diff
@@ -105,6 +104,7 @@ RDFT::RDFT(const std::shared_ptr<ngraph::Node>& op, const GraphContext::CPtr con
         auto signalSizesNode = ov::as_type<ov::op::v0::Constant>(op->get_input_node_ptr(2));
         if (!signalSizesNode)
             return;
```

Comment:
If Axes input is constant, but Signal size is not constant, the isAxesConstant will be false. That is not correct.

Reply:
done